pyproject.toml

```diff
@@ -0,0 +1,74 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "open-data-pvnet"
+version = "0.1.1"  # Static version
+readme = { file = "README.md", content-type = "text/markdown" }
+description = "An open-source project supporting solar forecasting with PVNet."
+license = { text = "MIT" }
+authors = [
+    { name="Peter Dudfield", email="info@openclimatefix.org" }
+]
+keywords = ["solar forecasting", "open-source", "renewable energy", "machine learning"]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Scientific/Engineering :: Atmospheric Science",
+]
+
+requires-python = ">=3.9"
+dependencies = [
+    "numpy",
+    "pandas",
+    "requests",
+    "xarray",
+    "zarr",
+    "h5netcdf",
+    "ocf_datapipes>=3.3.34",
+    "ocf_ml_metrics>=0.0.11",
```

Comment:
im not sure we need this?

Coudl we remove ocf_datapipes? We want to move over to ocf-data-sampler please. 
Perhaps we could just put `pvnet` in and that should cover this.